sample/pom.xml

```diff
@@ -34,6 +34,9 @@
     <modules>
         <module>rest-trivial</module>
         <module>rest</module>
+        <module>servlet-trivial</module>
+        <module>servlet-vaadin-v08</module>
```

Comment:
do we need 2 vaadin examples or only the v10 is enough? Since it is samples I'm quite tempted to only keep the last one, wdyt? Also, can it be tested (even if trivial) just to show a full sample?

Reply:
Good morning, well V8 and V10 are different. V8 with GWT and different API compared to V10 (based on Polymer). I can write tests for it, but wanted to see if the samples are welcome here ;-) I would like to add more and more samples, but maybe we can discuss this at a different place.

Reply:
Oh i see. Maybe send a mail on the dev list of openwebbeans, if you want to add a lot we can create another repo to keep the main one not too big.

And yes it is very appreciated to open project vision like that.